### PR TITLE
data:NikhilRaikwar

### PR DESCRIPTION
**Pull request by NikhilRaikwar**
## Fixes Issue

Adding my profile to LinkFree

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4643"></a>

### CODE REVIEW

File: data/NikhilRaikwar.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Nikhil Raikwar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Developer **OpenSource Enthusiast **. Technical Writer ",
+  "avatar": "https://github.com/NikhilRaikwar.png",
```
Comment (priyankarpal) [inline data/NikhilRaikwar.json:6]: Currently, you don't need to add an avatar code. You have to delete it.
``` suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
Looks good 🎉

File: data/NikhilRaikwar.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    {    
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/nikhilraikwar_18",
+      "icon": "FaInstagram"
+    },
+
+    {    
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/nikhilraikwar16/",
+      "icon": "FaLinkedIn"
```
Comment (Panquesito7) [inline data/NikhilRaikwar.json:28]: Wrong icon name.
```suggestion
      "icon": "FaLinkedin"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

**Comment by NikhilRaikwar** [comment]
Hey anyone can give me details when my profile is available on LinkFree

File: data/NikhilRaikwar.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Nikhil Raikwar",
+  "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline data/NikhilRaikwar.json:3]: As of the latest update, we no longer need `displayStatsPublic` property. So please remove this field from your json. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Comment by kumarsonsoff3** [comment]
Hey @NikhilRaikwar, not sure why did you close the PR when it was about to merge!?

**Comment by NikhilRaikwar** [comment]
Sorry for saying but I close this pr and Create a new pr and that pr is finally merge.

Thankyou for  your help.

**Pull request closed without merging**